.gitignore

```diff
@@ -1 +1,2 @@
 */*/work
+*.pyc
```

Comment:
I've added this to the plugins directly: https://github.com/opnsense/plugins/commit/24b90615

Thanks!